components/webfield/ProgramChairWithSeniorReviewerConsole.js

```diff
@@ -455,6 +456,279 @@ const Overview = ({ pcConsoleData }) => {
   )
 }
 
+const MessageReviewersModal = ({
```

Comment:
It seems like there's a lot of shared code between this and the MessageReviewersModal and the component used by the other consoles... if this is going to continue to be used by venues I wonder if there's a way to combine them. Possibly the MessageReviewersModal could have a param that takes a function that sends the emails, so you could just pass the function that starts on line 502 to the component.

Reply:
MessageReviewersModal is copied inside this console for easier cleanup so when these 2 conferences are over or when multiple reviewer groups are added in the venue request form only this console file need to be deleted.

i think message reviewers function only need to handle multiple reviewer groups, the email sending function should be the same.

Reply:
OK, so you are saying this feature of multiple reviewers groups will possibly be added to all the consoles and the  venue request form in the future.

I guess it makes sense to wait to modify MessageReviewersModal until then

Reply:
correct, reviewer groups will be an option in request form so all consoles will have it